packages/workbox-cli/src/app.js

```diff
@@ -18,6 +19,36 @@ const logger = require('./lib/logger');
 const readConfig = require('./lib/read-config');
 const runWizard = require('./lib/run-wizard');
 
+/**
+ * Runs the specified build command with the provided configuration.
+ *
+ * @param {Object} options
+ */
+async function runBuildCommand({command, config, watch}) {
+  try {
+    const {size, count, warnings} = await workboxBuild[command](config);
+
+    for (const warning of warnings) {
+      logger.warn(warning);
+    }
+
+    logger.log(`The service worker was written to ${config.swDest}\n` +
```

Comment:
I'm totally fine if we don't do this, but it might be nice have a slightly different log message in the case of a rebuild do to a watched file changing.